File: code/modul2/switchDemo.c
#include <stdio.h>
 
int main()
{
int n; 
printf("Masukkan angka 1-3: ");
scanf("%d", &n);
 
switch (n)
{
	case 1:
		printf("Anda memasukkan angka kecil");
	break;
	case 2:
		printf("Anda memasukkan angka sedang");
	break;
	case 3:
		printf("Anda memasukkan angka besar");
	break;
	default:
		printf("Anda memaskukkan angka yang salah");
	break;
}
printf("\n");
return 0;
}

File: code/octave/switchDemo.m
% caseDemo.m

n = input ('Masukkan angka dari 1 - 100 : ');

switch n
    case num2cell(1:33)
        disp ('Anda memasukkan angka kecil')
    case num2cell(34:67)
        disp ('Anda memasukkan angka medium');
    case num2cell(68:100)
        disp ('Anda memasukkan angka besar');
    otherwise
        disp ('And memasukkan angka yang salah');
end